tests/test_firefox_urls.py

```diff
@@ -37,11 +37,16 @@ class TestFirefoxURLs(object):
         '/firefox/tiles/',
         '/firefox/unsupported-systems/',
         '/firefox/unsupported/EOL/',
+        # Legacy URLs (Bug 1110927)
+        '/firefox/panorama/',
+        '/firefox/start/central.html',
+        '/firefox/sync/firstrun.html',
     ]
 
     def test_urls_ok(self, mozwebqa):
         for path in self.paths:
             url = mozwebqa.base_url + '/en-US' + path
             response = requests.get(url, timeout=3)
+            assert len(response.history) < 2
```

Comment:
Based on http://selenium.qa.mtv2.mozilla.com:8080/job/mozilla.org.prod.android-relnotes-pull/1/console perhaps this should be `<= 2`?